content/zh/docs/devops-user-guide/how-to-integrate/sonarqube.md

```diff
@@ -143,12 +143,12 @@ weight: 11310
 
    ![Webhook 页面信息](/images/docs/zh-cn/devops-user-guide/tool-integration/integrate-sonarqube-into-pipelines/webhook-page-info.png)
 
-### 步骤 4：将 SonarQube 配置添加到 ks-installer
+### 步骤 4：将 SonarQube 配置添加到 ks-devops
```

Comment:
If `devops-config` is to be edited in the following steps, I think we should change `ks-devops` to `devops-config`.

Reply:
See https://github.com/kubesphere/website/pull/2152/commits/aa4381a4b037e96bb37ee5834c1b16dea15203d1